config.json

```diff
@@ -47,7 +47,14 @@
                 "file_actions": true,
                 "aws_actions": true,
                 "preview_data": true,
-                "glue_crawler": true
+                "glue_crawler": true,
+                "show_stack_logs": "admin-only"
```

Comment:
The parameter `show_stack_logs` could be applied to any module that deploys a CloudFormation template upon module-object-creation. So in theory we could also add the `show_stack_logs` to the modules: ml_studio, notebooks and datapipelines. 
- important: what is the default value? does it change in comparison to the previous state for these modules?
- let's not forget to clearly explain this in the github pages :)

Reply:
@dlpzx, we don't use the MLstudio, notebooks, pipeline module and thus I didn't know there are stack logs for them. Let me check quickly how much is the effort. 

> important: we are not adding those modules in the map_target_to_config, what would happen to their stack logs?
That's a great point. For now, I can make it return None/ or some form of incorrect string so that `is_feature_allowe...` will throw and error. Also, in the error I can specify a way to implement this

> let's not forget to clearly explain this in the github pages :) 
Yes I will add those in the readme for sure.

Reply:
Redshift does not have an associated CDK stack; this is good and in case the parameter is not set it should default to enabled